src/Atlas2DSlice.ts

```diff
@@ -1,23 +1,148 @@
-import { BufferGeometry, Material, Mesh, PlaneGeometry, ShaderMaterial, ShapeGeometry } from "three";
-import { Volume } from ".";
-import { sliceFragmentShaderSrc, sliceVertexShaderSrc } from "./constants/volumeSliceShader";
-import RayMarchedAtlasVolume from "./RayMarchedAtlasVolume";
-import { rayMarchingShaderUniforms } from "./constants/volumeRayMarchShader";
+import { Box3, Box3Helper, BufferGeometry, Color, Group, LineBasicMaterial, Material, Matrix4, Mesh, PlaneGeometry, ShaderMaterial, ShapeGeometry, Vector2, Vector3 } from "three";
```

Comment:
I'm thinking autoformatting should have broken up this long line?

Reply:
I DID NOT KNOW THIS WAS A THING
I AM VERY EXCITED ABOUT THIS FEATURE